### PR TITLE
docs: provide link to the Github theme variables

### PR DESCRIPTION
**Pull request by NamNguyenInstructure**
Closes: INTUI-3548

### CODE REVIEW

File: packages/__docs__/src/Document/index.tsx [hunk 1]
```diff
@@ -199,6 +200,28 @@ class Document extends Component<DocumentProps, DocumentState> {
...
+
+    let legacySrcUrl
+
+    if (legacyGitBranch) {
+      legacySrcUrl = themeUrl.replace('/master/', `/${legacyGitBranch}/`)
+    }
+
+    return (
+      <View as="div" margin="0 0 x-large 0">
+        You can check these variables at{' '}
```
Comment (ke1k0) [inline packages/__docs__/src/Document/index.tsx:219]: Text suggestion: `See which global theme variables are mapped to the component here: [link]`

File: packages/__docs__/src/Document/index.tsx [hunk 2]
```diff
@@ -199,6 +200,28 @@ class Document extends Component<DocumentProps, DocumentState> {
...
+        <Link href={legacySrcUrl || themeUrl}>{themePath}</Link>
```
Comment (ke1k0) [inline packages/__docs__/src/Document/index.tsx:220]: I would also use this link in the block below on the "default theme variables" text.

Screenshot 2022-09-20 at 10 10 45

**Review by ke1k0 — CHANGES_REQUESTED** [review]

**Review by matyasf — APPROVED** [review]
nice!

**Review by ke1k0 — COMMENTED** [review]
Please merge the 2 commits into one (generally it is not nice to have "fix pr" commits littering the git history), and I think the commit type should be "docs" instead of "refactor".

File: packages/__docs__/buildScripts/utils/getPathInfo.ts [hunk 3]
```diff
@@ -23,14 +23,15 @@
...
+import fs from 'fs'
 import type { LibraryOptions, PackagePathData } from '../DataTypes'
 
 export function getPathInfo(
   resourcePath: string,
   projectRoot: string,
   library: LibraryOptions
 ): PackagePathData {
-  return {
+  const ans: PackagePathData = {
```
Comment (ke1k0) [inline packages/__docs__/buildScripts/utils/getPathInfo.ts:34]: I'd prefer more semantic names here, like `const: packagePathData: PackagePathData = { ... }`

**Review by ke1k0 — CHANGES_REQUESTED** [review]
Just had a small comment, otherwise looks good!

**Review by ke1k0 — APPROVED** [review]

**Pull request merged**